### DISCUSSION TITLE
Add `-q | --quiet` and `-qq | --quieter` options to the install script

### DISCUSSION THREAD

**Discussion by cesarcoatl** [starship/starship#3732] [Show and tell]
I've modified the `install.sh` script to add the `--quiet` and `--quieter` options.

Display `help`:

```bash
$ ./install/install.sh -h

install.sh [option]

Fetch and install the latest version of starship, if starship is already
installed it will be updated to the latest version.

Options
        -V, --verbose
                Enable verbose output for the installer

        -f, -y, --force, --yes
                Skip the confirmation prompt during installation

        -p, --platform
                Override the platform identified by the installer [default: pc-windows-msvc]

        -b, --bin-dir
                Override the bin installation directory [default: /usr/local/bin]

        -a, --arch
                Override the architecture identified by the installer [default: x86_64]

        -B, --base-url
                Override the base URL used for downloading releases [default: https://github.com/starship/starship/releases]

        -q, --quiet
                Quiet mode overrides verbosity

        -qq, --quieter
                Quieter mode overrides verbosity and uses force

        -h, --help
                Displays this help message
```

Passing `--quiet` does not display the steps for each shell, but still asks for user confirmation:

```bash
$ ./install/install.sh -q -b "$HOME/.starship/bin"

  Configuration
> Bin directory: /c/Users/thecesrom/.starship/bin
> Platform:      pc-windows-msvc
> Arch:          x86_64

> Tarball URL: https://github.com/starship/starship/releases/latest/download/starship-x86_64-pc-windows-msvc.zip
? Install Starship latest to /c/Users/thecesrom/.starship/bin? [y/N] y
> Installing Starship, please wait…
✓ Starship installed
```

Passing `--quieter` is similar passing `--quiet --force`, but it makes output more quiet:

```bash
$ ./install/install.sh -qq -b "$HOME/.starship/bin"

> Tarball URL: https://github.com/starship/starship/releases/latest/download/starship-x86_64-pc-windows-msvc.zip
> Installing Starship, please wait…
✓ Starship installed
```

Motivation

After upgrading multiple times I already have completed the steps for my shell, so there is no need to display the whole thing.

My shell scripting skills are not refined, but I've managed to hack my way into producing the results you see above.

**Comment by kidonng** [comment]
I'm not sure if installation script needs any silent flag, you don't run it on a regular basis and IMO it's always good to have info about installation.

**Reply by cesarcoatl** [reply]
I use Starship on Git BASH for Windows, and I do run the install script instead of relying on package managers like Homebrew or Chocolatey.

Running `curl -sS https://starship.rs/install.sh | sh` is still the "official" way of installing/upgrading, but I do use a custom `--bin-dir`, since the default location is locked.

This might be considered as an "advanced installation" method.

**Reply by kidonng** [reply]
I see, but isn't `>/dev/null` enough?

**Reply by cesarcoatl** [reply]
If I'm not mistaken, that may potentially silence errors. Wouldn't it?

**Reply by kidonng** [reply]
This only silences stdout, errors are stderr
